frontend/src/scenes/persons/cohortLogic.js

```diff
@@ -69,12 +69,14 @@ export const cohortLogic = kea({
                 cohort = await api.create('api/cohort', cohortFormData)
                 cohortsModel.actions.createCohort(cohort)
             }
+            breakpoint()
             delete cohort['csv']
             actions.setCohort(cohort)
             sharedListeners.pollIsFinished(cohort)
         },
-        checkIsFinished: async ({ cohort }) => {
+        checkIsFinished: async ({ cohort }, breakpoint) => {
             cohort = await api.get('api/cohort/' + cohort.id)
+            breakpoint()
             sharedListeners.pollIsFinished(cohort)
```

Comment:
Hmm... I guess unrelated to this PR, but this should be an action. 

TypeScript would complain here and you'll actually need to provide these arguments to the shared listener to have it work properly:

```
sharedListeners.pollIsFinished<A>(payload: A['payload'], breakpoint: BreakPointFunction, action: A, previousState: any) 
```